addons/crm/web_planner_data.xml

```diff
@@ -227,24 +218,26 @@
                         <li>Don't use stages to qualify opportunities, use tags instead</li>
                         <li>Every stage should be a step forward in the buyer's decision-making, not a task to carry out</li>
                     </div>
-
                     <p>Great pipelines have stages <strong>aligned with the buyer's buying process</strong>, not your selling process:</p>
                     <table class="table">
                         <thead>
                             <th></th>
-                            <th><span class="fa fa-thumbs-o-down"/> <strong> Bad</strong> Requirement</th>
-                            <th><span class="fa fa-thumbs-o-up"/> <strong> Good</strong> Requirement</th>
+                            <th width="50%"><strong>Example: Qualified stage</strong></th>
```

Comment:
by using the default width of <td> , the layout seems not good, so that i have set the width attribute of <td>. !! :)